specs/002-shared-ui-primitives/contracts/button.md

```
@@ -0,0 +1,110 @@
+| `btn` | Always |
+| `btn--primary` | When `variant() === 'primary'` |
+| `btn--secondary` | When `variant() === 'secondary'` |
+| `btn--loading` | When `loading() === true` |
+| `btn--disabled` | When `isDisabled() === true` |
+
+## Accessibility Contract
+
+| Attribute | Value | Condition |
+|-----------|-------|-----------|
+| `type` | `"button"` or `"submit"` | Always (from `type` input) |
+| `disabled` | present | When `loading() || disabled()` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)